framework/test/unit/modules/interoperability/internal_method.spec.ts

```diff
@@ -1655,6 +1919,8 @@ describe('Base interoperability internal method', () => {
 				),
 			).resolves.toBeUndefined();
 
+			// TODO [DONE]: Explicitly state: outboxKey = Buffer.concat([Buffer.from('83ed0d25', 'hex'), Buffer.from('0000', 'hex'), cryptoUtils.hash(OWN_CHAIN_ID)])
```

Comment:
I don't see that this was done

Reply:
I thought "Explicitly state" means a detailed comment. 

Now it is more explicit now